community_tasks/_template.py

```diff
@@ -33,7 +33,7 @@
 
 from lighteval.metrics import Metrics
 from lighteval.metrics.metrics import SampleLevelMetric
-from lighteval.metrics.utils import MetricCategory, MetricUseCase
+from lighteval.metrics.utils.utils import MetricCategory, MetricUseCase
```

Comment:
`metrics.utils.utils` is kinda ugly :sweat_smile: - we could put this file as an `init` in `metrics.utils` maybe

Reply:
I don't like this "abusing" of __init__ tbh especially when there are more files in that folder. I will rename the file to metric_utils

Reply:
But that's almost worst XD
metrics.utils.metrics_utils?